kpet/run.py

```diff
@@ -18,6 +18,18 @@
 from kpet import data
 
 
+class Case:
+    # pylint: disable=too-few-public-methods,too-many-instance-attributes
+    """A test case in a test suite"""
+    EXPORTED_ATTRIBUTES = ["name", "max_duration_seconds", "hostRequires",
+                           "partitions", "kickstart", "waived", "role",
+                           "environment", "maintainers"]
+
+    def __init__(self, case):
+        for attr in self.EXPORTED_ATTRIBUTES:
+            setattr(self, attr, getattr(case, attr))
```

Comment:
I don't like having lots of assignments when there isn't any transformation, so I did this loop instead. It makes it completely different to the other `kpet.run` classes, so I don't know if it's worth it, but hey.